src/crepe/system/AnimatorSystem.h

```diff
@@ -26,6 +24,9 @@ class AnimatorSystem : public System {
 	 * looping).
 	 */
 	void update() override;
+
+private:
+	LoopTimer & timer = LoopTimer::get_instance();
```

Comment:
LoopTimer moet toegevoegd worden aan `Mediator` en via `this->mediator` opgevraagd worden